### DISCUSSION TITLE
how to unsubscribe the github status notifications?

### DISCUSSION THREAD

**Discussion by LynnWonder** [community/community#57897] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I used to subscribe github status by my email and i want to unsubscribe it now, but how?

**Comment by nirav-gajera** [comment, marked as answer]
1. Open one of the GitHub status emails you received.

2. Scroll down to the bottom of the email. You should find a section that says "Unsubscribe" or "Manage your subscription."

3. Click on the "Unsubscribe" or "Manage your subscription" link. This will typically take you to a web page where you can manage your email preferences.

**Reply by LynnWonder** [reply]
got it

**Comment by ghost** [comment]
> 1. Open one of the GitHub status emails you received.
> 
> 
> 
> 2. Scroll down to the bottom of the email. You should find a section that says "Unsubscribe" or "Manage your subscription."
> 
> 
> 
> 3. Click on the "Unsubscribe" or "Manage your subscription" link. This will typically take you to a web page where you can manage your email preferences.

**Reply by ghost** [reply]
👍

**Comment by Bhavnoor9407** [comment]
Open any of GitHub emails you received.

Scroll down to the bottom of the email. You will find a section that says "Unsubscribe" or "Manage your subscription."

Click on the "Unsubscribe" or "Manage your subscription" link. It will take you to a web page where you can manage your email preferences.

**Comment by djabranemmd** [comment]
To unsubscribe from GitHub status emails, you have a few options:

1. Open any email you received from GitHub Status.
2. Scroll to the bottom of the email.
3. Click on “unsubscribe” or “manage subscriptions”.

Or you can do it directly:

* Go to: https://www.githubstatus.com/
* Scroll down and find “Subscribe to updates”
* Enter your email and you should see an option to manage or unsubscribe your subscription.

Also check your GitHub notification settings in your account:

* https://github.com/settings/notifications

From there you can disable status emails if they are linked to your account.
